src/lib.rs

```diff
@@ -34,7 +34,7 @@ gstd::metadata! {
 }
 
 #[no_mangle]
-pub unsafe extern "C" fn init() {
+unsafe extern "C" fn init() {
```

Comment:
Seems that our entry functions now can be safe, isn't it?
```suggestion
extern "C" fn init() {
```